packages/slider/src/Slide.tsx

```diff
@@ -74,9 +74,9 @@ class Slide extends React.PureComponent<SlideProps, SlideState> {
     beforeExitToNext: PropTypes.func,
     beforeExitToPrev: PropTypes.func,
     className: PropTypes.string,
-    didEnter: PropTypes.bool,
-    didEnterAsPrev: PropTypes.bool,
-    didEntereAsNext: PropTypes.bool,
+    didEnter: PropTypes.bool | PropTypes.func,
+    didEnterAsPrev: PropTypes.bool | PropTypes.func,
+    didEntereAsNext: PropTypes.bool | PropTypes.func,
```

Comment:
These actually should just be `PropTypes.func`. 

(For propTypes if we want to supply multiple types, then it would be `PropTypes.oneOfType([ PropTypes.bool, PropTypes.func ])`, but the `boolean` typing was a holdover from a previous implementation.

Reply:
Thank you! Didn't know that about multiple types.